packages/fether-react/src/Health/Health.js

```diff
@@ -54,29 +51,28 @@ class Health extends Component {
 
     const chainNameAppend = isLoading(chainName) ? '' : ` (${chainName})`;
 
-    switch (status) {
-      case STATUS.NO_NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. No nodes connected';
-      case STATUS.DOWNLOADING:
-        return `Downloading Parity Ethereum... (${payload.percentage}%)`;
-      case STATUS.LAUNCHING:
-        return 'Launching the node...';
-      case STATUS.NODE_CONNECTED_AND_NO_INTERNET:
-        return 'No internet. Connected to node.';
-      case STATUS.NO_CLOCK_SYNC:
-        return 'Clock of host not in sync';
-      case STATUS.NO_PEERS:
-        return 'No peer node connections';
-      case STATUS.SYNCING:
-        return `Syncing...${
-          payload && payload.percentage && payload.percentage.gt(0)
-            ? ` (${payload.percentage.toFixed(0)}%)`
-            : ''
-        }${chainNameAppend}`;
-      case STATUS.GOOD:
-        return `Synced${chainNameAppend}`;
-      default:
-        return JSON.stringify(payload); // Just in case payload is an object
+    if (!status.nodeConnected && !status.internet) {
+      return 'No internet. No nodes connected';
```

Comment:
By `nodeConnected`, I actually meant "is fether connected to our local light node". So the message here should be "No node connected" or "Not connected to light node"

Reply:
I think we should use the first one "No node connected" since they may choose to run a full node, and the second message may make them think there's something wrong with them doing that?

Reply:
Also, there's limited width for the title in the modal

Reply:
Sounds good